RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
+
+        }
+        private string name;
+
+        public string Name
+        {
+            get { return name; }
+            set
+            {
+                if (!value.Equals(""))
```

Comment:
This does not solve a case of entering a line full of whitespace.
Make yourself familiar and use the C# methods for strings.
This will save you a lot of time and bugs: https://learn.microsoft.com/en-us/dotnet/api/system.string?view=net-7.0
In this case the method IsNullOrWhiteSpace.

Reply:
fixed